x/deployment/types/msgs.go

```diff
@@ -2,107 +2,103 @@ package types
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-// MsgCreate defines an SDK message for creating deployment
-type MsgCreate struct {
-	Owner sdk.AccAddress `json:"owner"`
-	// Sequence uint64         `json:"sequence"`
-	// Version []byte      `json:"version"`
-	Groups []GroupSpec `json:"groups"`
+// msg types
+const (
+	TypeMsgCreateDeployment = "create_deployment"
+	TypeMsgUpdateDeployment = "update_deployment"
+	TypeMsgCloseDeployment  = "close_deployment"
```

Comment:
Can you make these values be the same as those in the [codec registration](https://github.com/ovrclk/akash/pull/464/files/fc49dbd9eca7cb378cf096094bc213d5e28d777a..1f149e55e87b155d142e93543cffa76859043f0b#diff-e5c9ba0cb12c3a49b6d1e97c98885903R15-R17) and use these constants there?

If they don't need to be exported, please make them private.